docs: Updates for accuracy in Add to Existing Repository page. (#9379)
vercel/turborepo: docs/repo-docs/getting-started/add-to-existing-repository.mdx

</Step>
<Step>
### Add a `packageManager` field to root `package.json`

> Turborepo optimizes your repository using information from your package manager. To declare which package manager you're using, add a [`packageManager`](https://nodejs.org/api/packages.html#packagemanager) field to your root `package.json` if you don't have one already.

Reviewer: Should we advertise [--dangerously-disable-package-manager-check](https://turbo.build/repo/docs/reference/run#--dangerously-disable-package-manager-check) here? I don't want to encourage people to use it, but if for some reason this is a blocker for people we should try to help them unblock themselves.

Author: Yes.

```diff title="package.json"
{
+  "packageManager": "npm@8.5.0"
}
```

</Step>
<Step>
### Run tasks with `turbo`

You can now run the tasks you added to `turbo.json` earlier using Turborepo. Using the example tasks from above:

```bash title="Terminal"
turbo build check-types
```

> This runs the `build` and `check-types` tasks at the same time, respecting the dependency graph of your project.

Reviewer: I feel like "respecting the dep graph of your project" is a little vague. Does focusing on "shared tasks" feel better to you? 

```suggestion
This runs the `build` and `check-types` tasks at the same time, reusing results from any shared tasks.
```

Author: Yes.
